### PR TITLE
Bd 1834 cf to model

### PR DESCRIPTION
**Pull request by yan130**
## Description
Use met.process to convert CF met to model.
in this case, 
 stage = (download.raw = FALSE, met2cf = FALSE, standardize = FALSE, met2model = TRUE)
met.process will pass "inputfiles" (pecan.nc or pecan.zip) to 
.met2model.local.module and convert to model specific file without write an entry to bety.

## Motivation and Context
https://opensource.ncsa.illinois.edu/jira/browse/BD-1556

## Note
failed to update man of data.atmosphere: 
Dependency package PEcAn.logger not available.

### CODE REVIEW

**Comment by tonygardella** [comment]
@robkooper Can you take a look at this one?

**Comment by tonygardella** [comment]
@robkooper Can you take a look at this one as well?

File: modules/data.atmosphere/R/met.process.R [hunk 1]
```diff
@@ -234,30 +249,47 @@ met.process <- function(site, input_met, start_date, end_date, model,
...
+                                            str_ns = str_ns,
+                                            site = site, 
+                                            start_date = start_date, end_date = end_date, 
+                                            browndog = browndog, 
+                                            new.site = new.site,
+                                            overwrite = overwrite$met2model,
+                                            exact.dates = reg.model$exact.dates,
+                                            spin = spin)
+      
+      model.id  <- met2model.result$model.id
```
Comment (robkooper) [inline modules/data.atmosphere/R/met.process.R]: is this section not needed later in the code?

File: modules/data.atmosphere/R/met.process.R [hunk 2]
```diff
@@ -109,13 +109,28 @@ met.process <- function(site, input_met, start_date, end_date, model,
...
+    if(input_met$source == "CUSTOM"){
+      # internal file is provided
+      stage <- list(download.raw = FALSE, met2cf = FALSE, standardize = FALSE, met2model = TRUE)
+      format.vars <- list()
+      format.vars$lat <- NULL
+      format.vars$lat <- NULL
+      format.vars$lat <- NULL
+    } else {
+      if(is.null(model)){
+        stage <- list(download.raw = TRUE, met2cf = TRUE, standardize = TRUE, met2model = FALSE)
```
Comment (robkooper) [inline modules/data.atmosphere/R/met.process.R]: you changed the code from original, is this correct to set met2model to TRUE/FALSE based model.

File: modules/data.atmosphere/R/met.process.R (hunk 1 above)
Comment (yan130) [reply modules/data.atmosphere/R/met.process.R]: used in the next line

File: modules/data.atmosphere/R/met.process.R (hunk 2 above)
Comment (yan130) [reply modules/data.atmosphere/R/met.process.R]: is.null(model) == true mean we don't need to convert to model format, we just need the internal format,

is.null(model) == false mean to do the convert old way, from XML to model.

**Comment by yan130** [comment]
cannot find the original branch, will create a new one.

**Pull request closed without merging**